Visual Studio 15.8.3 - Could not load file or assembly Clarius.TransformOnBuild.MSBuild.Task.dll

Since updating to Visual Studio 15.8.3 I can no longer build projects using Clarius.TransformOnBuild in Visual Studio.

This is the Build Output:

`1>CopyTaskAssemblyToTempFolder:
1>  Creating directory "C:\Users\MyUserName\AppData\Local\Temp\29c1cdd1-49b5-413d-a473-04f96c0be9bb".
1>  Copying file from "C:\\MyProject\packages\Clarius.TransformOnBuild.1.21.0\build\Clarius.TransformOnBuild.MSBuild.Task.dll" to "C:\Users\MyUserName\AppData\Local\Temp\29c1cdd1-49b5-413d-a473-04f96c0be9bb\Clarius.TransformOnBuild.MSBuild.Task.dll".
1>_CopyFilesMarkedCopyLocal:
1>  Touching "C:\\MyProject\MyProject\obj\Debug\MyProject.csproj.CopyComplete".
1>C:\\MyProject\packages\Clarius.TransformOnBuild.1.21.0\build\Clarius.TransformOnBuild.targets(43,5): error MSB4062: The "Clarius.TransformOnBuild.MSBuild.Task.TransformOnBuildTask" task could not be loaded from the assembly C:\Users\MyUserName\AppData\Local\Temp\3380110d-bf98-4c4f-b30c-663cadce97f1\Clarius.TransformOnBuild.MSBuild.Task.dll. Could not load file or assembly 'file:///C:\Users\MyUserName\AppData\Local\Temp\3380110d-bf98-4c4f-b30c-663cadce97f1\Clarius.TransformOnBuild.MSBuild.Task.dll' or one of its dependencies. The system cannot find the file specified. Confirm that the <UsingTask> declaration is correct, that the assembly and all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framework.ITask.`

--- a/nuget/build/Clarius.TransformOnBuild.targets
+++ b/nuget/build/Clarius.TransformOnBuild.targets
@@ -17,20 +17,9 @@
 <Project
   ToolsVersion="4.0"
   xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <TempFolder>$([System.IO.Path]::GetTempPath())$([System.Guid]::NewGuid())</TempFolder>
-  </PropertyGroup>
-  <Target
-    Name="CopyTaskAssemblyToTempFolder"
-    BeforeTargets="BeforeBuild">
-    <Copy
-      SourceFiles="$(MSBuildThisFileDirectory)Clarius.TransformOnBuild.MSBuild.Task.dll"
-      DestinationFolder="$(TempFolder)"
-    />
-  </Target>
   <UsingTask
     TaskName="Clarius.TransformOnBuild.MSBuild.Task.TransformOnBuildTask" 
-    AssemblyFile="$(TempFolder)\Clarius.TransformOnBuild.MSBuild.Task.dll"
+    AssemblyFile="$(MSBuildThisFileDirectory)\Clarius.TransformOnBuild.MSBuild.Task.dll"
   />
   <Target
     Name="TransformOnBuild"

